tools/run_tests/performance/bq_upload_result.py

```diff
@@ -29,51 +29,59 @@
 import massage_qps_stats
 
 gcp_utils_dir = os.path.abspath(
-    os.path.join(os.path.dirname(__file__), '../../gcp/utils'))
+    os.path.join(os.path.dirname(__file__), "../../gcp/utils"))
```

Comment:
The double quotes everywhere should be changed back to single quotes. Thanks!

Reply:
Right, I didn't realize that I have changed them,  fixed. Thank you!